R/utils_assert.R

```diff
@@ -15,6 +15,14 @@ assert_scalar_character <- function(x, name = deparse(substitute(x))) {
   assert_character(x, name)
 }
 
+assert_db_name <- function(x, name = deparse(substitute(x))) {
+  assert_character(x, name)
+  if (!grepl("^[[:lower:]_]+$", x)) {
```

Comment:
numbers are allowed too I think?  So `^[[:lower:_][[:lower:]_[:digit:]]*$` (or equivalently `^[a-z_][a-z0-9_]*$`)

Reply:
Fixed this too